src/Tribe/Context/locations.php

```diff
@@ -76,6 +76,25 @@
 	],
 	'post_type' => [
 		'read' => [
+			Tribe__Context::FUNC        => static function() {
+				$post_type_objs = get_post_types( [
+					'public' => true,
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Array double arrow not aligned correctly; expected 3 space(s) between "'public'" and double arrow, but found 1.
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "TRUE" but found "true"

<sub>hash: e9ac134d0d0ab3414abd88998191868d</sub>